<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>12</y>
     <width>644</width>
     <height>576</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>534</width>
        <height>86</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>China</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>United States</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>England</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>South Korea</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Male</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Female</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>558</width>
          <height>501</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Hello</string>
        </attribute>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>12</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Hello</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>39</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>It's me</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>558</width>
          <height>501</height>
         </rect>
        </property>
        <attribute name="label">
         <string>You Are Beatiful</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>My love is brilliant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>My love is prue</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Bilibili</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="image_label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resource.qrc">:/images/football.png</pixmap>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Google Translate</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="movie_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>122</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="scroll_area_btn">
      <property name="text">
       <string>Scoll Area</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tool_box_btn">
      <property name="text">
       <string>Tool Box</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tab_widget_btn">
      <property name="text">
       <string>Tab Widget</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
